README.md

```diff
@@ -1,165 +1,17 @@
-[日本語](#日本語デジタルテキスト組版の要件)は英文の後にあります。
-# Requirements for Japanese Digital Text Layout
+# Requirements for Japanese Digital Text Layout (日本語デジタルテキスト組版の要件)
 
-This Readme is a working document. We will update it along with the progress of the project.
+The [Japanese Language Enablement home page](https://w3c.github.io/jlreq/home) provides links and information about related work and the organization of the group. 
+「日本語組版の要件 (JLReq)」を含め、関連したプロジェクト、および日本語タスクフォースの構成、参加の仕方などの情報は[Japanese Language Enablement, 日本語組版要件のホームページ](https://w3c.github.io/jlreq/home)にあります。
```

Comment:
There should be a blank line to separate these two paragraphs, otherwise, the Markdown parser might think they're in the same paragraph.

Reply:
added as suggestion, and will merge into